### DISCUSSION TITLE
wezterm title bar doesn't use gnome theme

### DISCUSSION THREAD

**Discussion by Sherlock-Holo** [wezterm/wezterm#4423] [Q&A]
### What Operating System(s) are you running on?

Linux X11, Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

both of them using the GNOME mutter

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

when using GNOME x11, the wezterm title bar is using the gtk title bar with my GNOME theme, when I switch to GNOME wayland, the title bar become wezterm's title bar

I prefer my GNOME theme title bar, and also I know in wayland, wayland  compositor won't create the window decorator, the program should do this job itself

is there any way to let wezterm use GNOME theme title bar, for example create a wrapper to wrap the wezterm?

**Comment by wez** [comment]
See: https://github.com/wez/wezterm/issues/1969
